solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -240,6 +240,66 @@ client.add(Arrays.asList(d1, d2));
 ====
 ======
 
+=== ScalarQuantizedDenseVectorField
+Because dense vectors can have a costly storage footprint, it may be worthwhile to use a technique called "quantization"
```

Comment:
I would specify that's not only strictly storage (because actually the storage footprint on disk increases with quantisation).

I would stress memory/latency benefits above all.